### ISSUE TITLE
[aardvark] Management UI insanely slow browsing collection with big documents

### ISSUE THREAD

**Issue by abunet** [arangodb/arangodb#14172]
## Environment

* __ArangoDB Version__:        3.7.10

* __Deployment Mode__:       Single Server

__Problem__:
The management ui tries to fetch first 25 fields of a document when browsing a collection in the listview.
If one of the first 25 fields is a blob with large data, the effect is that paging the list of the collection results insanely slow

The result is that the management UI is unusable for browsing documents in the collection

__Suggestion__:
Introduce a per collection option that allows to specify a list of fields to fetch in the collection.
One simpler alternative could be an option (per collection) that asks to fech only the _key field
This should be applied only in the listview, clicking the document I will espect the retrieve of the full document

![problem](https://user-images.githubusercontent.com/804576/117819547-ef3d1500-b269-11eb-9237-9802cefc4dde.png)

**Comment by dothebart** [comment]
can you please specify the size of these documents?

**Comment by abunet** [comment]
about 70MB each document. for example
```
{
  "_key": "key",
  "name": "my name",
  "attachments" [
         {
             "id: "1",
             "name": "my title",
            "data": "my large blob encoded64hexstring "
         },
  ]
}
```

**Comment by abunet** [comment]
HI any news about this? 
I've seen that is marked as a feature but it is actually blocking the management of arango. 
It is not possible to manage the indexes of the collection because the first thing that is done opening the collection is the attempt to download the data, completely blocking the interface....

**Comment by dothebart** [comment]
sorry no news regarding this.
Please note that you also can use arangosh or other drivers to manipulate indices. 
With arangosh you could do it like this:

```
127.0.0.1:8529@_system> db._create('test')
[ArangoCollection 663, "test" (type document, status loaded)]
127.0.0.1:8529@_system> db.test.ensureIndex({type: "persistent", fields: [ "a" ], sparse: true });
{ 
  "deduplicate" : true, 
  "fields" : [ 
    "a" 
  ], 
  "id" : "test/683", 
  "inBackground" : false, 
  "isNewlyCreated" : true, 
  "name" : "idx_1711510417857052672", 
  "sparse" : true, 
  "type" : "persistent", 
  "unique" : false, 
  "code" : 201 
}
```
Please see [the documentation on creating indices](https://www.arangodb.com/docs/stable/indexing-working-with-indexes.html#creating-an-index) for more details.

**Comment by palashkaria** [comment]
@abunet thanks for the report, and apologies for the long wait on this.

Now that the UI is being worked on more actively, this issue has been fixed (& back-ported to 3.10/3.11). From now, you should see the 'Info' tab as the default selected one when you visit a collection

Here's the PR with the fix: https://github.com/arangodb/arangodb/pull/18761

Once again, I really appreciate you taking out the time to report this!
